bundle/manifests/dpu-operator.clusterserviceversion.yaml

```diff
@@ -414,6 +414,10 @@ spec:
                   value: quay.io/openshift/dpu-intel-ipu-vsp:latest
                 - name: IntelVspP4Image
                   value: quay.io/openshift/dpu-intel-ipu-p4sdk:latest
+                - name: MarvellVspImage
+                  value: quay.io/openshift/dpu-marvell-vsp:latest
+                - name: IntelNetSecVspImage
```

Comment:
Ah I see it is there now.
Maybe we need a guide to onboard VSP images because there maybe more DPUs in the future that would need a VSP.

Reply:
Or a shell/python/make command.

Reply:
This comment shouldn't block this PR.

Reply:
I think now that the exact URLs are there, it's gonna be easier.

Imagine you add a new environment variable. You cargo cult what is already and will grep for one of the existing URLs. Then you find all the locations and just add them there too.

It should be reasonably hard to miss for something that happens very infrequently.

Also, more important seems actual tests that the Operatorhub image work... those should also catch such a bug.